concent-vm/install-golem.yml

```diff
@@ -0,0 +1,54 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../concent-builder/repositories.yml
+    - vars.yml
+  tasks:
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:   Clone golem repository
+          git:
+            repo:    "{{ repositories.golem.url }}"
+            dest:    "{{ golem_dir }}"
+            version: "{{ golem_version }}"
+            clone:   yes
+            update:  yes
+
+        - name:   Create virtualenv for golem
+          command:  virtualenv ~/golem_virtualenv --python python3.6 creates="~/golem_virtualenv"
+
+        - name:   Install requirements for golem
+          pip:
+            requirements:       "{{ golem_dir }}/{{ item }}"
+            virtualenv:         ~/golem_virtualenv
+            virtualenv_python:  python3.6
+          with_items:
+            - requirements.txt
+            - requirements-test.txt
+
+        - name:   Run setup.py develop
+          shell: ~/golem_virtualenv/bin/python setup.py develop
+          args:
+            chdir: "{{ golem_dir }}"
+
+        - name:   Download hyperg
+          get_url:
+            url:      "{{ golem_hyperdrive_repository }}"
+            dest:     ~/hyperg.tar.gz
+            mode:     0644
+            checksum: "{{ golem_hyperdrive_checksum }}"
+
+        - name:   Extract hyperg
+          unarchive:
+            src:        ~/hyperg.tar.gz
+            dest:       ~/
+            remote_src: yes
+
+        - name:    Copy to execute path
+          file:
+            src:  ~/hyperg/hyperg
+            dest: ~/bin/hyperg
```

Comment:
Please reword: `Copy to execute path` -> `Install a symbolic link to hyperg in ~/bin/`